CRM/Report/Form/Event/Income.php

```diff
@@ -382,4 +375,17 @@ public function postProcess() {
     parent::endPostProcess();
   }
 
+  /**
+   * Get statuses with the counted filter set to TRUE.
+   *
+   * @return array
+   */
+  protected function getActiveParticipantStatuses() {
+    $activeParticipantStatuses = [];
+    foreach (CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label") as $id => $label) {
+      $activeParticipantStatuses[$id] = $label;
+    }
```

Comment:
Why not `$activeParticipantStatuses = CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label");`  or `return  CRM_Event_PseudoConstant::participantStatus(NULL, "is_counted = 1", "label");`

Reply:
Thanks @pradpnayak on looking into it your suggestions make sense & improve readability - I have updated the PR